### ISSUE TITLE
Documentation Microsite

### ISSUE THREAD

**Issue by ChristopherDavenport** [ChristopherDavenport/cormorant#3]
Get the documentation microsite up and operational including examples. 

Sections for the core operations and then pages for each of the modules.

**Comment by calvinbrown085** [comment]
@ChristopherDavenport 
```
Quick Start

To use log4cats in an existing SBT project with Scala 2.11 or a later version, add the following dependencies to your build.sbt depending on your needs:
``` 

I think you might have had a mis-type here

**Comment by ChristopherDavenport** [comment]
This is what you get when you copy-paste. Yes, I'll fix that.

**Comment by ChristopherDavenport** [comment]
Fixed by #35
